### DISCUSSION TITLE
How to query the last ever data input by the user

### DISCUSSION THREAD

**Discussion by KA-devl** [supabase/supabase#11255] [Questions]
Hey, i would like to query the last data input by a user X.
Context : Currently building a fitness app, i need to show the user his "most recent" workout. One way to do this is by fetching all his workouts and get the last element of the array. However this will affect the performance since querying a bunch of workouts is a bad idea. 
How can i query this one "most recent" workout pushed in the database by the user?

In brief : how to query the last (or top most) element in a table?
Thank you!

**Comment by GaryAustin1** [comment]
You would use order by on either a created_at time column (descending) or an incrementing id column (descending) and then use limit 1.
